docs/catalog/cloud/use-circuit-breaker.md

```diff
@@ -0,0 +1,43 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: cloud
+description: [PATTERN_DESCRIPTION_METADATA]
+tags: 
+ - compute
+ - size:medium
+---
+
+# Use circuit breaker patterns
+
+## Description
+
+Modern applications need to communicate with other applications on a regular basis. These other applications have their own deployment schedule, downtimes and availability or the network connection to these application might have problems.
```

Comment:
i think in these situations where we are referencing collections like the Azure Well Architected framework, we can use similar or paraphrased wordings in the description/problem and solution areas.

that said, i think this description could be a bit more clear at addressing the problem that includes the following pulled from Azure's [documentation](https://learn.microsoft.com/en-us/azure/architecture/patterns/circuit-breaker):
`In these situations it might be pointless for an application to continually retry an operation that is unlikely to succeed, and instead the application should quickly accept that the operation has failed and handle this failure accordingly.`

Reply:
We usually do only 1-2 sentences in the description of the patterns for v1. Including all mentioned would make a book out of it. Also we do not balantly copy from other documentation, as the license is unclear.

I think there is no place to draw the connection, otherwise we would have to copy the other paragraphs.

Reply:
I think the sentence, that is already there: "If the other application is not reachable, all network requests against this other application will fail and future network requests are futile." 
does cover the part that you pasted:
"In these situations it might be pointless for an application to continually retry an operation that is unlikely to succeed, and instead the application should quickly accept that the operation has failed and handle this failure accordingly."